open-craft/opencraft
# BB-1877: Build themed components for some pages

This PR adds two components used on the domains and instance configuration page, as well and theming fixes and improvements on multiple components.

**Testing instructions:**
1. Create a file `frontend/.env` with the following contents:
```
REACT_APP_ENTERPRISE_COMPARISON_LINK=https://www.opencraft.com
REACT_APP_CONTACT_US_LINK=https://www.opencraft.com
```
1. `npm install && npm start`
2.  Open the registration form, and check that the banner about Enterprise instances is working and follows the mockups.
3. Click on the buttons and check that the links are correctly passed.
3. Enter a random text on the domain name, go to the next step of the registration process and check that the "Your domain is connected" windows follows the mockups.
4. Check that tests pass.

**Notes:**
The code still needs some cleanup.
Mockups on Zeplin linked on the ticket.

**Reviewer:**
